RevenueCatUI/CustomerCenter/ViewModels/CustomerCenterViewModel.swift

```diff
@@ -83,19 +83,35 @@ import RevenueCat
         self.purchasesProvider = purchasesProvider
     }
 
+    #if DEBUG
```

Comment:
Nit: I might put this inside a fileprivate extension to separate it from the production code

Reply:
It's used in CustomerCenterView as well :/